src/sagemaker/mxnet/README.rst

```diff
@@ -31,13 +31,68 @@ In the following sections, we'll discuss how to prepare a training script for ex
-| WARNING                                                                                                                       |
-+===============================================================================================================================+
-| This required structure for training scripts will be deprecated with the next major release of MXNet images.                  |
-| The ``train`` function will no longer be required; instead the training script must be able to be run as a standalone script. |
-| For more information, see `"Updating your MXNet training script" <#updating-your-mxnet-training-script>`__.                   |
-+-------------------------------------------------------------------------------------------------------------------------------+
++--------------------------------------------------------------------------------------------------------------------------------------------------+
+| WARNING                                                                                                                                          |
++==================================================================================================================================================+
+| The structure for training scripts changed with MXNet version 1.3.                                                                               |
+| Please make sure you are referring to the correct section of this README when preparing your script.                                             |
+| To jump to how to upgrade an old script to the new format, see `"Updating your MXNet training script" <#updating-your-mxnet-training-script>`__. |
++--------------------------------------------------------------------------------------------------------------------------------------------------+
+
+For versions 1.3 and higher
+'''''''''''''''''''''''''''
+Your MXNet training script must be a Python 2.7 or 3.5 compatible source file.
+
+The training script is very similar to a training script you might run outside of SageMaker, but you can access useful properties about the training environment through various environment variables, such as
+
+* ``SM_MODEL_DIR``: A string representing the path to the directory to write model artifacts to.
+  These artifacts are uploaded to S3 for model hosting.
+* ``SM_NUM_GPUS``: An integer representing the number of GPUs available to the host.
+* ``SM_OUTPUT_DATA_DIR``: A string representing the filesystem path to write output artifacts to. Outut artifacts may include checkpoints, graphs, and other files to save, not including model artifacts.
+  These artifacts are compressed and uploaded to S3 to the same S3 prefix as the model artifacts.
+
+Supposing two input channels, 'train' and 'test', were used in the call to the MXNet estimator's ``fit`` method, the following will be set, following the format "SM_CHANNEL_[channel_name]":
```

Comment:
Here's my attempt at restructuring this:

* ``SM_CHANNEL_XXXX``:  A string that represents the path to the directory that contains the input data for the specified channel. For example, if two input channels, named 'train' and 'test', are specified in the call to the MXNet estimator's ``fit`` method, the environment variables ``SM_CHANNEL_TRAIN`` and ``SM_CHANNEL_TEST`` are set.

Reply:
I like that a lot better. I changed the second sentence a little bit but added the extra bullet point and removed the paragraph/separate list